src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class Ball {
+
+    private Color color;

```

Comment:
color should be string
https://github.com/mate-academy/jv-lottery/blob/master/checklist.md#use-name-for-getting-string-representation-of-enum-constants
```suggestion
    private String color;
```